src/libraries/System.Runtime/tests/System/Runtime/DependentHandleTests.cs

```diff
@@ -0,0 +1,73 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Xunit;
+
+namespace System.Runtime.Tests
+{
+    // NOTE: DependentHandle is already heavily tested indirectly through ConditionalWeakTable<,>.
+    // This class contains some specific tests for APIs that are only relevant when used directly.
```

Comment:
I think we should still have a reasonable set of tests for DependentHandle directly covering its basic functionality.  Its use in ConditionalWeakTable is an implementation detail.

Reply:
Added some more unit tests for the available public APIs (getting/setting target/dependent, and liveness tests) in https://github.com/dotnet/runtime/pull/54246/commits/3314d7e7713e0cb306e416253611a765be668fbe, happy to add more if there's additional specific scenarios you think we should also check in particular! 🙂